.zenodo.json

```diff
@@ -60,6 +65,9 @@
       "name": "Hayot-Sasson, Val\u00e9rie",
       "orcid": "0000-0002-4830-4535"
     },
+    {
+      "name": "James Kent"
```

Comment:
@jdkent Do you want to add your ORCID/affiliation?

Reply:
thanks for the heads up, yes, I'll open a pull request against this branch.